src/frontend/App/hooks/useSetValgtPersonIdent.ts

```diff
@@ -0,0 +1,15 @@
+import { useApp } from '../context/AppContext';
+import { useEffect } from 'react';
+import { IPersonopplysninger } from '../typer/personopplysninger';
+
+// eslint-disable-next-line
+export const useSetValgtPersonIdent = (personopplysninger: IPersonopplysninger) => {
```

Comment:
funker det med `usePersonIdent = ...` ?

Reply:
Hmm ja, mulig, tenkte det kunde være fint att denne kun ble brukt for å sette `valgtPersonIdent`, då det er litt hack å bruke `use` og ikke returnere noe.
Men kan endre